### PR TITLE
Edit mask on phone number field of volunteer signup dialog allows for creation of malformed input

### LINKED ISSUE #497: Edit mask on phone number field of volunteer signup dialog allows for creation of malformed input
The following screenshots show some examples of the malformed phone number inputs that are allowed by the current edit mask.

screenclip

screenclip 1

screenclip 2

### PR DESCRIPTION
**Pull request by mmoore99**
Fixes #497 

Add jquery.maskedinput plugin to correct current edit mask errors and provide more complete, non hard coded edit mask capability.

### CODE REVIEW

**Comment by tonysurma** [comment]
@MisterJames can someone look over this PR before I merge?

File: AllReadyApp/Web-App/AllReady/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -129,6 +130,10 @@
         <script src="//d3js.org/d3.v3.min.js"
                 asp-fallback-src="~/lib/d3/d3.js">
         </script>
+        <script src="//cdnjs.cloudflare.com/ajax/libs/jquery.maskedinput/1.4.1/jquery.maskedinput.min.js"
+                asp-fallback-src="~/lib/jquery.maskedinput/dist/jquery.maskedinput.js"
+                asp-fallback-test="window.mask">
```
Comment (mheggeseth) [inline AllReadyApp/Web-App/AllReady/Views/Shared/_Layout.cshtml]: Fallback test should be `window.jQuery && window.jQuery.fn && window.jQuery.fn.mask`. Loading `jquery.maskedinput` doesn't do anything to  `window.mask`.

**Comment by mmoore99** [comment]
Submitted change as suggested by @mheggeseth. Thanks, Mike.

**Comment by mheggeseth** [comment]
Merge it. :shipit:

**Pull request merged**